src/harmony/Link/Link.module.css

```diff
@@ -0,0 +1,28 @@
+/* View */
```

Comment:
Current links have 0.2s transition for color change on hover. Don't know if we still need that in new design